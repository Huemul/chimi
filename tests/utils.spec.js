const { listDependencies } = require('../lib/utils')

describe('utils', () => {
  describe('listDependencies', () => {
    const dependencies = [
      'trae',
      { module: 'es6-promise' },
      { name: '_', module: 'lodash' },
      { name: 'config', module: './config', type: 'let' },
    ]

    it('should list dependencies in different types', () => {
      expect(listDependencies(dependencies)).toMatchSnapshot()
    })
  })
})
