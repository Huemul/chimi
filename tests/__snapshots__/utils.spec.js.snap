// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils listDependencies should list dependencies in different types 1`] = `
"const trae = require('trae');
require('es6-promise');
const _ = require('lodash');
let config = require('./config');"
`;
